.github/ISSUE_TEMPLATE/syncer-support.md

```diff
@@ -6,6 +6,34 @@ about: "Requesting support for syncer Tool"
 
 ## syncer Support
 
+Before submitting your issue, please check the requirements list below (all need to be met):
```

Comment:
we would use tool to check these

Reply:
yes, but we have not implements this.
so, remove these?

Reply:
we can create a more detail issue template for debug deep into debug